lib/app_funcs.sh

```diff
@@ -14,27 +14,17 @@ function restore_app() {
 
 
 function copy_hex() {
-  mkdir -p ${build_path}/.mix/archives
-  mkdir -p ${build_path}/.hex
-
-
-  # hex is a directory from elixir-1.3.0
-  full_hex_file_path=$(ls -dt ${HOME}/.mix/archives/hex-* | head -n 1)
```

Comment:
Finding out the full_hex_file_path isn't necessary; setting `MIX_HOME` and `HEX_HOME` means it will end up in the cache and in the build slug since Mix will install hex using `MIX_HOME`.